### PR TITLE
Message Repository UI

### PR DESCRIPTION
**Pull request by Gawshan987**
Hey in this PR, I have setup the message repo with lucky and the next steps are:
- create the database 
- create the graph QL
- integrate the backend into the feature 
image

### CODE REVIEW

**Review by LuckyHariharan — COMMENTED** [review]
Hey great work on this feature, looks great! For some reason it is not deploying on vercel. I think its that useAuth hook error we saw. Curiously addGoal has that same error but works on the page.  Let's investigate more together and/or ask a coach/Vithushan during our next meeting.

**Comment by LuckyHariharan** [comment]
@Gawshan987  make sure to remove console.logs and update your description. otherwise great work on the UI portion, and after this feel free to put up your graphql progress in your next

**Comment by LuckyHariharan** [comment]
awesome work! looks good to me!

**Comment by Gawshan987** [comment]
@vithushan19 This is ready for review.

File: graphql/studentPortal/messageRepo/insertMessage.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { gql } from "@apollo/client";
+
+export const insert_Message = gql`
+mutation MyMutation($reachout_message: String = "", $date: timestamp = "") {
+    insert_message_repository_one(object: {reachout_message: $reachout_message, date: $date}) {
+      reachout_message
+      date
+    }
+  }
+
```
Comment (vithushan19) [inline graphql/studentPortal/messageRepo/insertMessage.ts:10]: Delete these empty lines to make the code cleaner to read.

**Review by vithushan19 — APPROVED** [review]
Nice lgtm

File: graphql/studentPortal/messageRepo/insertMessage.ts (hunk 1 above)
Comment (Gawshan987) [reply graphql/studentPortal/messageRepo/insertMessage.ts]: Okay will do

**Pull request merged**